src/picknik_ur_base_config/config/base_config.yaml

```diff
@@ -90,17 +90,17 @@ hardware:
       # To use parameters from a different package, place them in a config/ROBOT_NAME/ directory,
       # replace ROBOT_NAME with the value used for hardware.type in this file.
       - kinematics_parameters_file:
-      # Load default_kinematics.yaml from ur_description/config/ur5e
+      # Load default_kinematics.yaml from ur_description/config/<ur_type>
           package: "ur_description"
-          path: "config/ur5e/default_kinematics.yaml"
+          path: "config/%>> hardware.type <<%/default_kinematics.yaml"
```

Comment:
Does this `%>> blah <<%` syntax actually work? I wasn't aware of this, nor do I see it in the source?